changelogs/7.0.asciidoc

```diff
@@ -3,17 +3,66 @@
 
 https://github.com/elastic/apm-server/compare/6.7...7.0[View commits]
 
-// * <<release-notes-7.0.0>>
+* <<release-notes-7.0.0>>
 * <<release-notes-7.0.0-rc2>>
 * <<release-notes-7.0.0-rc1>>
 * <<release-notes-7.0.0-beta1>>
 * <<release-notes-7.0.0-alpha2>>
 * <<release-notes-7.0.0-alpha1>>
 
-////
 [[release-notes-7.0.0]]
 === APM Server version 7.0.0
-////
+
+https://github.com/elastic/apm-server/compare/6.7...7.0[View commits]
+
+These release notes include all changes made in the alpha, beta, and RC releases of 7.0.0.
+
+Also see:
+
+* {apm-server-ref-v}/breaking-changes.html[Breaking changes in 7.0]
+
+[float]
+==== Added
+
+- Update Elastic APM Go agent to v1.1.2 {pull}1711[1711], {pull}1728[1728].
+- Update response format of the healthcheck handler and prettyfy all JSON responses {pull}1748[1748].
+- Add transaction.type to error data {pull}1781[1781].
+- Rename transaction.span_count.dropped.total to transaction.span_count.dropped {pull}1809[1809].
```

Comment:
i don't know how it was reflected on previous releases or if it was discussed previously, but i'd say that L30, L31, L32, L35 and L36 are breaking changes

Reply:
They were in the added column for `7.0.0-beta1`. Moved to breaking changes